server/analytics/__init__.py

```diff
@@ -16,9 +16,13 @@
 from analytics.track_activity import TrackActivityResource, TrackActivityService
 from analytics.processed_items_report import ProcessedItemsResource, ProcessedItemsService
 from analytics.content_quota_reports import ContentQuotaReportResource, ContentQuotaReportService
+from superdesk.metadata.utils import add_aggregation
 
 
 def init_app(app):
+    add_aggregation('items_over_days', {'date_histogram': {'field': '_updated', 'interval': 'day'}})
```

Comment:
It is my understanding of the `Aggregation Management` that `add_aggregation` adds this aggregation globally. I think it might be better to add this aggregation before running the elastic search query, then removing this aggregation after the query has completed.

Reply:
This is no longer required because of the use of the `Aggregations Manager` when running the reports